tags.html

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: "Tags"
+description: "Tag cloud from all the posts"
+permalink: /tags
+sitemap: false
+---
+
+<div class="tags-page text-center">
+  <h1 class="tags-page-header">
+    Tags
+  </h1>
+
```

Comment:
Would it be possible to fix the indentations on this page please?

Reply:
done. There was a tabs/spaces mix up in the file. I converted them all to spaces.